scripts/ci/linux/build.sh

```diff
@@ -13,11 +13,18 @@ echo "PLATFORM_CFLAGS += $CUSTOMFLAGS" >> libs/openFrameworksCompiled/project/li
 cd libs/openFrameworksCompiled/project
 make Debug
 
-echo "Building emptyExample"
```

Comment:
> building empty example makes the build crash

unit tests [also make the build crash](https://travis-ci.org/openframeworks/openFrameworks/jobs/75839351#L1501), so you just replaced one build-crashing application with another...